airo-camera-toolkit/airo_camera_toolkit/cameras/realsense/realsense.py

```diff
@@ -59,6 +59,12 @@ def __init__(
             depth_resolution = Realsense.RESOLUTION_720 if fps <= 30 else Realsense.RESOLUTION_480
             config.enable_stream(rs.stream.depth, depth_resolution[0], depth_resolution[1], rs.format.z16, fps)
 
+        # Avoid having to reconnect the USB cable
```

Comment:
@m-decoster  could you maybe link to the page where you found the solution? Could be useful later on